lib/pltr/v2/selectors/ui.js

```diff
@@ -236,6 +241,43 @@ export const orientationSelector = createSelector(
   }
 )
 
+export const stickyHeaderCountSelector = createSelector(
+  pinnedPlotlinesSelector,
+  timelineViewIsStackedSelector,
+  hierarchyLevelCount,
+  selectedOrientationSelector,
+  (pinnedPlotlines, timelineViewIsStacked, hierarchyLevelCount, selectedOrientation) => {
+    if (selectedOrientation == 'vertical') {
+      return 1
+    }
+    if (timelineViewIsStacked) {
+      return hierarchyLevelCount
+    } else if (pinnedPlotlines) {
+      return pinnedPlotlines + 1
+    } else {
+      return 1
+    }
+  }
+)
+
+export const stickyLeftColumnCountSelector = createSelector(
+  pinnedPlotlinesSelector,
+  timelineViewIsStackedSelector,
+  hierarchyLevelCount,
+  selectedOrientationSelector,
+  (pinnedPlotlines, timelineViewIsStacked, hierarchyLevelCount, selectedOrientation) => {
+    if (selectedOrientation == 'horizontal') {
+      return 1
+    } else if (timelineViewIsStacked) {
```

Comment:
Same question here about the stacked view.

Reply:
Oh never mind!
In stacked view, you can't flip the timeline so this will never be a problem.